### PR TITLE
VOTE-500: improve styling with utilities

### PR DESCRIPTION
**Pull request by mlloydbixal**

### CODE REVIEW

File: src/components/CardInfo.jsx
Comment (mlloydbixal) [inline src/components/CardInfo.jsx:1]: I deleted this component because it is not being reused

File: src/components/CardSelect.jsx
Comment (mlloydbixal) [inline src/components/CardSelect.jsx:1]: I deleted this component because it is not in the design

File: src/components/PathSelection.jsx
Comment (mlloydbixal) [inline src/components/PathSelection.jsx:1]: I was not able to style the border correctly using utilities, it might need to be done during custom styling.

**Review by rayestrada — COMMENTED** [review]
@mlloydbixal  can you audit the css files and remove unused styles and empty css files? Also please resolve the merge confilct.

**Review by rayestrada — APPROVED** [review]

**Pull request merged**